cupy_setup_build.py

```diff
@@ -511,39 +519,65 @@ def get_long_description():
 
 
 def prepare_wheel_libs():
-    """Prepare shared libraries for wheels.
+    """Prepare shared libraries and include files for wheels.
 
     On Windows, DLLs will be placed under `cupy/cuda`.
-    On other platforms, shared libraries are placed under `cupy/_libs` and
+    On other platforms, shared libraries are placed under `cupy/.data/lib` and
     RUNPATH will be set to this directory later.
+    Include files are placed under `cupy/.data/include`.
     """
-    libdirname = None
     if PLATFORM_WIN32:
-        libdirname = 'cuda'
+        lib_dirname = 'cuda'
         # Clean up existing libraries.
-        libfiles = glob.glob('cupy/{}/*.dll'.format(libdirname))
+        libfiles = glob.glob('cupy/{}/*.dll'.format(lib_dirname))
         for libfile in libfiles:
             print('Removing file: {}'.format(libfile))
             os.remove(libfile)
     else:
-        libdirname = '_lib'
+        lib_dirname = '.data/lib'
         # Clean up the library directory.
-        libdir = 'cupy/{}'.format(libdirname)
+        libdir = 'cupy/{}'.format(lib_dirname)
         if os.path.exists(libdir):
             print('Removing directory: {}'.format(libdir))
             shutil.rmtree(libdir)
-        os.mkdir(libdir)
 
-    # Copy specified libraries to the library directory.
-    libs = []
-    for lib in cupy_setup_options['wheel_libs']:
+    include_dirname = '.data/include'
+
+    # Collect files to copy
+    files_to_copy = []
+
+    # Library files
+    lib_base_path = os.path.join('cupy', lib_dirname)
+    for srcpath in cupy_setup_options['wheel_libs']:
+        relpath = os.path.basename(srcpath)
+        dstpath = path.join(lib_base_path, relpath)
+        files_to_copy.append((
+            srcpath,
+            dstpath,
+            path.join(lib_dirname, relpath)))
+
+    # Include files
+    include_base_path = os.path.join('cupy', include_dirname)
+    for include_path_spec in cupy_setup_options['wheel_includes']:
+        srcpath, relpath = include_path_spec.split(':')
```

Comment:
```suggestion
        srcpath, relpath = include_path_spec.rsplit(':', 1)
```

Reply:
Oh I forgot Windows can have `:` in paths.

Reply:
FYI, on Windows `:` is invalid in path, whereas it's allowed in Linux/Mac.